index.html

```diff
@@ -22,10 +22,65 @@
                   <li><a href="">Contact</a></li>
                   <li><a href=""><img class="" src="image/bg-envlope.png" alt="menu icon"></a></li>
               </ul>
-              <ul class="menu-icons"><li><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
+              <ul class="menu-icons">
+                <li>
+                  <a href="#" id="mbm-icon">
+                    <img class="menuimg" src="image/menu.png" alt="menu icon">
+                  </a>
+                </li>
+              </ul>
           </nav>
       </header>
       <main class="main-cont">
+
+        <div class="mbmenu-hide-show" id="hide-show" >
+          <article class="header-section-mbm" id="mbm-article">
+            <div class="header-inner-cont-mbm">
+              <section class="inner-section">
+                <div class="cncel-cont">
+                  <div class="cncel">
+                    <a href="#" class="mbmenu-li-a">
+                      <img src="image/close-Icon.png" id="button" alt="close icon">
+                    </a>
+                  </div>
+                </div>
+                <div class="menu-list-cont">
+                  <div class="mblist-cont">
+                    <ul class="mbmenu-ul">
+                      <li mbm-list-item>
+                        <a href="#" class="mbmenu-li-a">
+                          <span class="li-span">Portfolio</span> 
+                          <span>
+                            <img src="image/arrow-right.png" alt="riht arrow">
+                          </span>
+                        </a>
+                      </li>
+                      <li>
+                        <a href="#" class="mbmenu-li-a">
+                          <span class="li-span">About</span>
+                          <span>
+                            <img src="image/arrow-right.png" alt="riht arrow">
+                          </span>
+                        </a>
+                      </li>
+                      <li>
+                        <a href="#contact-section" class="mbmenu-li-a">
+                          <span class="li-span">Contact</span>
+                          <span>
+                            <img src="image/arrow-right.png" alt="riht arrow">
+                          </span>
+                        </a>
+                      </li>
```

Comment:
- [ ] You did a really nice job working on the mobile menu! 💯 However, there's a feature that hasn't been implemented yet. According to the requirements:
![image](https://user-images.githubusercontent.com/77069011/203534656-e5277fe1-9f84-4693-8afb-82050351a3fa.png)

At the moment, when I click on the `Portfolio` or `About` links, the heading section is displayed. 🤔 Please consider implementing this feature in all your links the same way you did with the `Contact` one, this will provide a nice navigation ⛵ to your page and, in consequence, a better user experience! 🤓 🛠️ 

💡 _Hint: you can use the `"href"` attribute of the `<a>` tag to target the `id` of the `<section>` you want to display, the same as you did with your `Contact` link._